.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: CI
```

Comment:
Ideally, in the future try to keep these kinds of changes in separate PRs so they are more focused.

Reply:
Makes sense :+1: Will do.